### PR TITLE
Use same code on docker-compose and kubernetes

### PR DESCRIPTION
**Pull request by rajat-soda**
**What type of PR is this?**
Fixes the following. 
 - Build Latest images for multi-cloud . 
 - Use correct path for initdb.sh
 - Use same code on both Kubernetes and Docker-Compose.
 -
image (15)
image (16)

> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
Fixes the following. 
 - Build Latest images for multi-cloud . 
 - Use correct path for initdb.sh
 
**Which issue(s) this PR fixes**:

Fixes #
#1215 
#1223 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED

**Test Report**:
on docker-compose : 

image (15)

On Kubernetes
image (16)

**Special notes for your reviewer**:

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -121,7 +121,7 @@ services:
       - ./s3/pkg/datastore/yig/conf/:/etc/yig
       - /etc/ceph:/etc/ceph
       - ./examples/multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
-      - ./s3/pkg/conf/tidb.sql:/etc/config/tidb/s3.sql
+      - ./s3/pkg/conf/tidb.sql:/etc/config/tidb/tidb.sql
```
Comment (kumarashit) [inline docker-compose.yml:124]: will this require changes in initdb.sh? Or else, S3 functionalities will be impacted

Comment (rajat-soda) [reply docker-compose.yml:124]: initdb.sh already has the change.

Comment (kumarashit) [reply docker-compose.yml:124]: ok

File: dataflow/pkg/scheduler/lifecycle/lifecycle_sched.go [hunk 2]
```diff
@@ -33,14 +28,18 @@ import (
...
-	s3 "github.com/opensds/multi-cloud/s3/proto"
 	osdss3 "github.com/opensds/multi-cloud/s3/proto"
+	s3 "github.com/opensds/multi-cloud/s3/proto"
 	log "github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
+	"os"
+	"sort"
+	"strconv"
+	"sync"
+	"time"
```
Comment (AmitRoushan) [inline dataflow/pkg/scheduler/lifecycle/lifecycle_sched.go:39]: as per convention, core packages should be on top of file

File: api/cmd/main.go [hunk 3]
```diff
@@ -32,11 +32,13 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const (
-	serviceName = "soda.multicloud.v1.api"
-)
-
 func main() {
+
+	serviceName := "api"
```
Comment (AmitRoushan) [inline api/cmd/main.go:37]: Please use ```constants``` block to define constants.
Please also check rest of the code.

File: block/pkg/service/service.go [hunk 4]
```diff
@@ -44,9 +44,18 @@ type blockService struct {
 func NewBlockService() pb.BlockHandler {
 
 	log.Infof("Init block service finished.\n")
+
+	blkService := "block"
+	backendService := "backend"
+
+	if(os.Getenv("MICRO_ENVIRONMENT") == "k8s"){
```
Comment (AmitRoushan) [inline block/pkg/service/service.go:51]: Please use ```k8s``` as constant.
Also check rest of code

**Review by AmitRoushan — CHANGES_REQUESTED** [review]

File: dataflow/pkg/scheduler/lifecycle/lifecycle_sched.go (hunk 2 above)
Comment (rajat-soda) [reply dataflow/pkg/scheduler/lifecycle/lifecycle_sched.go:39]: even after go fmt, its still in bottom.

File: api/cmd/main.go (hunk 3 above)
Comment (rajat-soda) [reply api/cmd/main.go]: done

File: block/pkg/service/service.go (hunk 4 above)
Comment (rajat-soda) [reply block/pkg/service/service.go]: done.

File: dataflow/pkg/scheduler/lifecycle/lifecycle_sched.go (hunk 2 above)
Comment (AmitRoushan) [reply dataflow/pkg/scheduler/lifecycle/lifecycle_sched.go:39]: Please refer : https://github.com/golang/go/wiki/CodeReviewComments#imports

**Review by AmitRoushan — APPROVED** [review]
Please log comment as issue:
```as per convention, core packages should be on top of file.
Please refer : https://github.com/golang/go/wiki/CodeReviewComments#imports
```

LGTM

**Comment by PravinRanjan10** [comment]
LGTM

File: api/pkg/dataflow/service.go [hunk 5]
```diff
@@ -28,12 +28,19 @@ import (
...
 	s3 "github.com/opensds/multi-cloud/s3/proto"
 	log "github.com/sirupsen/logrus"
+	"os"
 )
 
 const (
-	backendService  = "soda.multicloud.v1.backend"
-	s3Service       = "soda.multicloud.v1.s3"
-	dataflowService = "soda.multicloud.v1.dataflow"
+	MICRO_ENVIRONMENT = "MICRO_ENVIRONMENT"
```
Comment (sushanthakumar) [inline api/pkg/dataflow/service.go:35]: Suggestion: Many service names looks to be reused, can think of moving to common file in future

**Review by kumarashit — APPROVED** [review]
LGTM

**Comment by sushanthakumar** [comment]
LGTM

**Comment by rajat-soda** [comment]
Opened issue #434 to track these suggestions and naming convention changes.

**Comment by kumarashit** [comment]
These conditional service naming is ok for now. Please open issue and track it to fix properly going ahead

**Comment by rajat-soda** [comment]
Opened #435 to track the conditional naming

**Review by NajmudheenCT — APPROVED** [review]
LGTM

**Pull request merged**